src/containers/facility/GroupBooking/RoomCardGroup.tsx

```diff
@@ -0,0 +1,52 @@
+import type { RoomTypes } from '@windingtree/glider-types/dist/win';
+import type { OfferRecord } from 'src/store/types';
+import { Box, Divider, Grid, Stack, TextField, Typography } from '@mui/material';
+import { useFormContext } from 'react-hook-form';
+import { RoomInformation } from '../RoomInformation';
+
+export interface FacilityGalleryProps {
+  offer: OfferRecord;
+  room: RoomTypes;
+}
+
+export const RoomCardGroup: React.FC<{
+  room: RoomTypes;
+  offer: OfferRecord;
+  index: number;
+}> = ({ offer, room, index }) => {
+  const { register } = useFormContext();
+  const price = `${offer.price?.currency} ${Number(offer.price?.public).toFixed(2)}`;
```

Comment:
`Number(undefined).toFixed(2)` produced `NaN`. I guess you need add fallback to zero in such case